docs/faq/rancher-is-no-longer-needed.md

```diff
@@ -55,11 +54,3 @@ To detach the cluster,
 3. Click **Delete**.
 
 **Result:** The registered cluster is detached from Rancher and functions normally outside of Rancher.
-
-## What if I don't want my RKE cluster or hosted Kubernetes cluster managed by Rancher?
```

Comment:
Similar to the above I think "RKE cluster" here means something provisioned by Rancher.